api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1133,35 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: Subscription is a high level consumption unit.
+    properties:
+      application_id:
+        type: string
+        example: 'price-service'
+        description: The id of application that will consume from the subscription.
+      event_types:
+        type: array
+        items:
+          type: string
+        description: |
+          EventTypes to subscribe to.
+          The order is not important. Subscriptions that differ only be the order of EventTypes will be considered
+          the same and will have the same id.
+          * Currently only subscription to a single EventType is supported. Subscriptions with more than one EventType
+          in event_types property will be rejected.
+      use_case:
```

Comment:
I don't have a strong opinion here, but I think that having a name like "something_id" will make our users think that they actually need to get this id from somewhere. And having it without "id" may provoke entering just a text description. So each approach has its pros and cons.